src/App.vue

```diff
@@ -109,22 +58,19 @@ export default {
       try {
         const { data } = await this.$airtable.get(`/programs`, {
           params: {
-            filterByFormula: `AND('${window.sitecoreItemId}' = ARRAYJOIN({locationGUID}), OR({endDate} >= TODAY(), {endDate} = ''))`,
+            filterByFormula: `AND({group} != '', '${window.sitecoreItemId}' = ARRAYJOIN({locationGUID}), OR({endDate} >= TODAY(), {endDate} = ''))`,
```

Comment:
@JordanPisani I filtered the records directly out of the results using Airtable's syntax.